modules/rosa-adding-tags-cli.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-adding-tags-cli{context}"]
+= Adding tags to a machine pool using the ROSA CLI
+
+You can add tags to a machine pool for your {product-title} cluster by using the ROSA CLI.
+
+.Prerequisites
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a {product-title} (ROSA) cluster.
+
+.Procedure
+
+. You can add tags to new or existing machine pools:
+* Create a machine pool with a custom tag by running the following command:
++
+--
+[source,terminal]
+----
+$ rosa create machinepools --cluster=<name> --replicas=<replica_count> \
+     --tags=<key>=<value>,<effect>,<key>=<value>,<effect> \ <1>
+----
+<1> Replace `<key>=<value>,<effect>,<key>=<value>,<effect>` with a key, value, and effect for each tag, for example `--tags=key1=value1:NoSchedule,key2=value2:NoExecute`. Available effects include `NoSchedule`, `PreferNoSchedule`, and `NoExecute`.This list overwrites any modifications made to node tags on an ongoing basis.
```

Comment:
```suggestion
<1> Replace `<key>=<value>,<effect>,<key>=<value>,<effect>` with a key, value, and effect for each tag. For example `--tags=key1=value1:NoSchedule,key2=value2:NoExecute`. Available effects include `NoSchedule`, `PreferNoSchedule`, and `NoExecute`. This list overwrites any modifications made to node tags on an ongoing basis.
```
This is a lot of info in a footnote. Is this the best way to capture all of that.